src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -119,6 +119,9 @@ class HaConfigIntegrations extends SubscribeMixin(HassRouterPage) {
             }
           });
           if (!newEntries.length && !fullUpdate) {
+            if (!this._configEntries) {
```

Comment:
This feels like a patch for symptom, not the cause. Instead of this, I think that we should change line 91, `let fullUpdate = this._configEntries === undefined;`. Then the existing logic works and this change can be removed

Reply:
I've changed and tested it. Thank you!